ParseUI-Widget/src/main/java/com/parse/ParseQueryAdapter.java

```diff
@@ -725,11 +661,12 @@ private View getDefaultView(Context context) {
   }
 
   private int getPaginationCellRow() {
-    return objects.size();
+    return getObjects().size();
   }
 
   private boolean shouldShowPaginationCell() {
-    return paginationEnabled && objects.size() > 0 && hasNextPage;
+    ParseQueryPager<T> pager = getPager();
+    return paginationEnabled && pager.getObjects().size() > 0 && pager.hasNextPage();
```

Comment:
Nit, since we have `getObjects()` helper method, let's just use the helper method so it is easy for us to change in the future.